package.json

```diff
@@ -80,7 +80,7 @@
     "hammerjs": "2.0.8",
     "intl": "1.2.5",
     "minimatch": "3.0.4",
-    "moment": "2.20.1",
+    "moment": "2.15.2",
```

Comment:
I  guess you can restore the 2.20.1 version, this old version has a minor security problem

Reply:
nope the ntegration/base_ver_2_app/package.json has to never change you can modify the script that update that project https://github.com/Alfresco/alfresco-ng2-components/blob/master/scripts/test-e2e-bc.sh in this script we keep trace of all the dependencies that need to be updated since the version 2.0.0 thanks

Reply:
Ok, thank you, I'll do that!